tests/test_validators.py

```diff
@@ -756,7 +757,7 @@ def repeat_b(cls, v):
             return v * 2
 
         @root_validator
-        def root_validator(cls, values):
+        def example_root_validator(cls, values):
```

Comment:
I noticed the namespace "conflict" (I ran into the issue below when I tried to move the decorator after the function definition). It doesn't need to be changed, but figured it was safe. I can change it back if desired.